pytest-embedded-idf/pytest_embedded_idf/dut.py

```diff
@@ -127,3 +129,88 @@ def close(self) -> None:
         if not self.skip_check_coredump:
             self._check_coredump()
         super().close()
+
+    Case = namedtuple('Case', 'index name type keyword group attribute subcase')
+
+    def get_test_menu(
+        self, regex: str = "Here's the test menu, pick your combo:(.+)Enter test for running.", trigger: str = ''
```

Comment:
all the `expect` functions could have 3 data types: `str`, `re.Pattern` and list of other types. How about we keep the same API as the expect functions?

```suggestion
        self, pattern = "Here's the test menu, pick your combo:(.+)Enter test for running.", trigger: str = ''
```